workspaces/orchestrator/plugins/orchestrator-form-react/package.json

```diff
@@ -35,6 +35,7 @@
   "dependencies": {
     "@backstage/core-components": "^0.15.1",
     "@backstage/core-plugin-api": "^1.10.0",
+    "@backstage/types": "^1.1.1",
```

Comment:
I missed this in the review
Can you please revert this change?

Reply:
I can't pass the prettier tests without this change

Reply:
I'll take a look to understand why
Meanwhile leave it as is